pkg/plugins/policies/meshproxypatch/api/v1alpha1/validator.go

```diff
@@ -204,15 +205,23 @@ func validateResourceValue(path validators.PathBuilder, value *string, res proto
 	return verr
 }
 
-func validatePatchValue(path validators.PathBuilder, value *string, res proto.Message) validators.ValidationError {
+func validatePatch(path validators.PathBuilder, value *string, jsonPatches []common_api.JsonPatchBlock, res proto.Message) validators.ValidationError {
 	var verr validators.ValidationError
+
 	if value != nil {
-		if err := mesh.ValidateAnyResourceYAMLPatch(*value, res); err != nil {
-			verr.AddViolationAt(path, fmt.Sprintf("native Envoy resource is not valid: %s", err.Error()))
+		if len(jsonPatches) > 0 {
```

Comment:
should this be `> 1`? Given the "must have only one" error. In which case it should be "must have a maximum of one".